scripts/deploy.memecoin.mjs

```diff
@@ -52,11 +53,23 @@ const randomMemecoin = {
 
 const ledgerCanisterId = await deployLedger(randomMemecoin);
 
-await deployIndex({
-	...randomMemecoin,
-	ledgerCanisterId
-});
+const args = process.argv.slice(2);
+const withIndex = args?.find((arg) => ['--with-index'].includes(arg)) !== undefined;
+
+let indexCanisterId;
+
+if (withIndex) {
+	indexCanisterId = await deployIndex({
```

Comment:
`const indexCanisterId = withIndex ? ... : undefined ` maybe?

or you need it variable?

Reply:
Sure. Done.